utils/google_utils.py

```diff
@@ -6,40 +6,50 @@
 import platform
 import time
 from pathlib import Path
+import torch
 
 
 def attempt_download(weights):
     # Attempt to download pretrained weights if not found locally
     weights = weights.strip().replace("'", '')
-    msg = weights + ' missing, try downloading from https://drive.google.com/drive/folders/1Drs_Aiu7xx6S-ix95f9kNsA6ueKRpN2J'
-
-    r = 1  # return
-    if len(weights) > 0 and not os.path.isfile(weights):
-        d = {'yolov3-spp.pt': '1mM67oNw4fZoIOL1c8M3hHmj66d8e-ni_',  # yolov3-spp.yaml
-             'yolov5s.pt': '1R5T6rIyy3lLwgFXNms8whc-387H0tMQO',  # yolov5s.yaml
-             'yolov5m.pt': '1vobuEExpWQVpXExsJ2w-Mbf3HJjWkQJr',  # yolov5m.yaml
-             'yolov5l.pt': '1hrlqD1Wdei7UT4OgT785BEk1JwnSvNEV',  # yolov5l.yaml
-             'yolov5x.pt': '1mM8aZJlWTxOg7BZJvNUMrTnA2AbeCVzS',  # yolov5x.yaml
-             }
-
-        file = Path(weights).name
-        if file in d:
-            r = gdrive_download(id=d[file], name=weights)
-
-        if not (r == 0 and os.path.exists(weights) and os.path.getsize(weights) > 1E6):  # weights exist and > 1MB
-            os.remove(weights) if os.path.exists(weights) else None  # remove partial downloads
-            s = 'curl -L -o %s "storage.googleapis.com/ultralytics/yolov5/ckpt/%s"' % (weights, file)
-            r = os.system(s)  # execute, capture return values
-
-            # Error check
-            if not (r == 0 and os.path.exists(weights) and os.path.getsize(weights) > 1E6):  # weights exist and > 1MB
+    file = Path(weights).name
+
+    msg = weights + ' missing, try downloading from https://github.com/ultralytics/yolov5/releases/'
+    models = ['yolov5s.pt', 'yolov5m.pt', 'yolov5l.pt', 'yolov5x.pt']  # available models
+
+    if file in models and not os.path.isfile(weights):
+        # Google Drive
+        # d = {'yolov5s.pt': '1R5T6rIyy3lLwgFXNms8whc-387H0tMQO',
+        #      'yolov5m.pt': '1vobuEExpWQVpXExsJ2w-Mbf3HJjWkQJr',
+        #      'yolov5l.pt': '1hrlqD1Wdei7UT4OgT785BEk1JwnSvNEV',
+        #      'yolov5x.pt': '1mM8aZJlWTxOg7BZJvNUMrTnA2AbeCVzS'}
+        # r = gdrive_download(id=d[file], name=weights) if file in d else 1
+        # if r == 0 and os.path.exists(weights) and os.path.getsize(weights) > 1E6:  # check
+        #    return
+
+        try:  # GitHub
+            url = 'https://github.com/ultralytics/yolov5/releases/download/v2.0/' + file
```

Comment:
>https://github.com/ultralytics/yolov5/releases/download/v2.0/

Shouldn't this be v3? I'm pretty sure I'm getting a 

>torch.nn.modules.module.ModuleAttributeError: 'Detect' object has no attribute 'm'

error because of this.

Reply:
https://github.com/ultralytics/yolov5/blob/ffe9eb42389038972d47eecd44c0f0dc9f2cf033/utils/google_utils.py#L31-L35

Reply:
Oh, I checked out the branch tagged with v3.0 release thinking the most recent code was akin to a nightly release or something.

Reply:
Yes, I suppose that's a good analog. Things are still changing often as we sort through bugs and try to introduce new features, so for the moment the 'nightly' will be your best bet.